pages/signup.tsx

```diff
@@ -1,3 +1,163 @@
+import Link from "next/link";
+import Head from "next/head";
+import "twin.macro";
+import { useForm } from "react-hook-form";
+import { UserPlus } from "phosphor-react";
+import {
+    Button,
+    ButtonProps,
+    Container,
+    Grid,
+    Text,
+    Heading,
+    Divider,
+    IconButton,
+    VStack,
+    HStack,
+    Image,
+} from "@chakra-ui/react";
+import { Box } from "@chakra-ui/core";
+
+import { useScreen } from "hooks";
+import { InputField } from "components/units/InputField";
+
+const data = [
+    { name: "Google", Icon: Image, source: "/images/google.png" },
+    { name: "Facebook", Icon: Image, source: "/images/facebook.png" },
+    { name: "Twitter", Icon: Image, source: "/images/twitter.png" },
+    { name: "LinkedIn", Icon: Image, source: "/images/linkedin.png" },
+] as const;
+
+const SocialSignUp = () => {
+    const { min, max } = useScreen();
+
+    const buttonProps: ButtonProps = {
+        w: "full",
+        size: "lg",
+        fontSize: "sm",
+    } as const;
+
+    return (
+        <VStack justify="start" spacing={8}>
+            <Grid tw="w-full gap-4" autoFlow={["column", "row"]} as="ul">
+                {data.map(({ name, Icon, source }) => (
+                    <li key={name}>
+                        {max`sm` && (
+                            <IconButton
+                                {...buttonProps}
+                                aria-label={name}
+                                icon={
+                                    <Image src={source} style={{ height: "35px", width: "auto" }} />
+                                }
+                            />
+                        )}
+                        {min`sm` && (
+                            <Button {...buttonProps} aria-label={name}>
+                                <HStack justify="start" spacing={8}>
+                                    <Image src={source} style={{ height: "35px", width: "auto" }} />
+                                    <Text>Sign Up with {name}</Text>
+                                </HStack>
+                            </Button>
+                        )}
+                    </li>
+                ))}
+            </Grid>
+        </VStack>
+    );
+};
+
+type FormData = {
+    email: string;
+    username: string;
+    password: string;
+    term: boolean;
+};
+
+const ManualFormSignUp = () => {
+    const { register, handleSubmit, errors } = useForm<FormData>();
+    const onSubmit = (userData: any) => {
+        console.log(userData);
```

Comment:
Make sure to remove console logs before merge

Reply:
We can remove all console.log references using Babel plugin http://localhost:8080/#/monitoring